src/Sonarr.Api.V3/Indexers/ReleaseController.cs

```diff
@@ -127,15 +127,15 @@ public async Task<object> DownloadRelease([FromBody] ReleaseResource release)
 
                         if (episodes.Empty())
                         {
-                            throw new NzbDroneClientException(HttpStatusCode.NotFound, "Unable to parse episodes in the release");
+                            throw new NzbDroneClientException(HttpStatusCode.NotFound, "Unable to parse episodes in the release, will need to be manually provided");
```

Comment:
```suggestion
                            throw new NzbDroneClientException(HttpStatusCode.NotFound, "Unable to parse episodes in the release, manual input interaction will be required.");
```

Reply:
Going to stick with the existing because I don't think `manual input interaction will be required.` is more clear as to what they need to do, but we can revisit down the road.